meshtastic/interfaces/ble/connection.py

```
@@ -359,6 +657,224 @@ def __init__(
         self.state_lock = state_lock
         self.thread_coordinator = thread_coordinator
 
+    def _dispatch_public_or_underscore(
```

Ruff (0.15.5)

[warning] Too many return statements (7 > 6)

(PLR0911)